test/mcse.jl

```diff
@@ -1,25 +1,41 @@
-@testset "mcse.jl" begin
-    samples = randn(100)
-
-    @testset "results" begin
-        result = @inferred(mcse(samples))
-        @test result isa Float64
-        @test result > 0
-
-        for method in (:imse, :ipse, :bm)
-            result = @inferred(mcse(samples; method=method))
-            @test result isa Float64
-            @test result > 0
-        end
-    end
+using Test
+using Distributions
+using MCMCDiagnosticTools
+using Statistics
+using StatsBase
 
-    @testset "warning" begin
-        for size in (51, 75, 100, 153)
-            @test_logs (:warn,) mcse(samples; method=:bm, size=size)
+@testset "mcse.jl" begin
+    @testset "estimand is within interval defined by MCSE estimate" begin
```

Comment:
For `mean`, `std`, and `quantile`, these tests are somewhat redundant with those for `ess_rhat`, since only difference is that for `ess_rhat` we compute the exact asymptotic variance of the estimator since we know the stationary distribution, whereas for `mcse` we estimate the asymptotic variance of the estimator. If we want to speed up our test suite, it may be better to remove the corresponding `ess_rhat` tests and use these as integration tests.

Reply:
I've opted to speed up our test suite by decreasing the number of replicates (params) for each suite to 100 instead of 1,000.